15_Ansible_test/setup_admin.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Create user admin, SSH and make
+  hosts: remote
+  vars_files:
+        - a_var.yml
+  tasks:
+        - name: Group 'wheel' creat
+          ansible.builtin.group:
+                 name: wheel
+                 state: present
+
+        - name: Allow 'wheel' group to have passwordless sudo
+          lineinfile:
+               dest: /etc/sudoers
+               state: present
+               regexp: '^%wheel'
+               line: '%wheel ALL=(ALL) NOPASSWD: ALL'
+               validate: visudo -cf %s
+        - name: Build user
+          ansible.builtin.user:
+               name: admin
+               groups: "wheel"
+               password: "{{ pass|password_hash('sha512') }}"
+        - name: SSH Key delivery
+          authorized_key:
+               user: admin
+               key: "{{ lookup('file', '/home/admin/.ssh/ansible_admin.pub') }}"
+               state: present
+               exclusive: True 
```

Comment:
The task was to create roles and use them in playbooks instead of writing all tasks in a playbook

Reply:
This is old files from a first Ansible Homework without roles.
All files for a Homework with roles located in DevOps_BigData/15_Ansible_test/ansible_roles_test/